### PR TITLE
Improve Accept Activities on Follow Requests

### PR DESCRIPTION
**Pull request by pfefferle**
This is mostly for debugging purposes

Improves #523 

## Proposed changes:

* A CLI Command to re-send Accept Activities when Follow request is stuck
* Schedule `Accept` Activity to prevent race conditions

### CODE REVIEW

File: includes/class-cli.php [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+		$follower = esc_url( $assoc_args['follower'] );
+		$follower = Followers::get_follower( $user->get__id(), $follower );
+
+		if ( ! $follower ) {
+			WP_CLI::error( __( 'Unknown Follower', 'activitypub' ) );
+		}
+
+		// get inbox
+		$inbox = $follower->get_shared_inbox();
```
Comment (mattwiebe) [inline includes/class-cli.php:47]: Only shared inbox? Or does the function fall back?

**Review by mattwiebe — COMMENTED** [review]
This looks nice and well worth a try.

I'll have to change the command on dotcom from `activitypub` to something else, but that's totally fine.

**Comment by pfefferle** [comment]
But does not solve this issue 😞 I have opened an issue on the Mastodon repo https://github.com/mastodon/mastodon/issues/27458

File: includes/class-cli.php (hunk 1 above)
Comment (pfefferle) [reply includes/class-cli.php:47]: it falls back to the inbox. Maybe we should rename it to something like `preferred_inbox`?!?

**Pull request closed without merging**